### PR TITLE
updated charts documentation

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
- use common templates for charts documentation
- updated OCI documentation

### CODE REVIEW

File: charts/victoria-metrics-auth/README.md [hunk 1]
```diff
@@ -15,73 +14,90 @@ Victoria Metrics Auth - is a simple auth proxy and router for VictoriaMetrics.
...
+
+    ```console
+    helm install vma oci://ghcr.io/victoriametrics/helm-charts/victoria-metrics-auth -f values.yaml -n NAMESPACE
+    ```
 
 Get the pods lists by running this commands:
 
 ```console
-kubectl get pods -A | grep 'vmauth'
+kubectl get pods -A | grep 'agent'
```
Comment (f41gh7) [inline charts/victoria-metrics-auth/README.md:80]: Looks like a copy-paste typo for all grep 'agent' occurrences.

**Review by f41gh7 — DISMISSED** [review]
LGTM

please fix possible typo

File: charts/victoria-metrics-auth/README.md (hunk 1 above)
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-auth/README.md]: thanks, fixed

**Review by f41gh7 — APPROVED** [review]
LGTM

**Pull request merged**